contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -6038,4 +6070,29 @@ remove_db_name_in_schema(const char *object_name, const char *object_type)
 	pfree(splited_object_name);
 
 	return (const char *)pstrdup(object_name + prefix_len);
-}
\ No newline at end of file
+}
+
+static int32
+tsql_get_numeric_typmod_from_exp(Plan *plan, Node *expr)
+{
+	int32       result_typmod = -1;
+	uint8_t     scale,
+                precision;
+	if (pltsql_protocol_plugin_ptr && *pltsql_protocol_plugin_ptr && (*pltsql_protocol_plugin_ptr)->get_numeric_typmod_from_exp)
```

Comment:
if (*pltsql_protocol_plugin_ptr && (*pltsql_protocol_plugin_ptr)->get_numeric_typmod_from_exp)
should be enough, right?

Reply:
Yes, updated.